packages/vue/src/components/organisms/SfTabs/_internal/SfTab.vue

```diff
@@ -17,8 +17,13 @@
     </slot>
     <div class="sf-tabs__content">
       <div v-if="isActive" class="sf-tabs__content__tab">
-        <!--@slot Default. Here you should pass your tab content -->
-        <slot />
+        <SfScrollable v-if="maxContentHeight">
+          <!--@slot Default. Here you should pass your tab content -->
+          <slot />
+        </SfScrollable>
+        <div v-else>
```

Comment:
please remove this `div`, you can use v-else directly on <slot />

Reply:
done.